opentracing/span.py

```diff
@@ -129,24 +129,26 @@ def set_tag(self, key, value):
         """
         return self
 
-    def log_event(self, event, payload=None):
-        """Logs an event to the span, with the current timestamp.
+    def log_kv(self, key_values, timestamp=None):
+        """Adds a log record to the Span.
 
-        :param event: an event name as a string
-        :param payload: an arbitrary structured payload. Implementations may
-            choose to ignore none, some, or all of the payload.
-        :return: returns the span itself, for chaining the calls
-        """
-        return self
+        For example,
 
-    def log(self, **kwargs):
-        """Records a generic Log event at an arbitrary timestamp.
+            span.log_kv({
+                "event": "time to first byte",
+                "packet_size": packet.size()})
```

Comment:
For illustration purposes I'd suggest using labels that cannot be simply passed as `**kwargs`, like `packet.size`